draft-ietf-quic-transport.md

```diff
@@ -884,20 +886,20 @@ is not excessive buffering at multiple layers.
 
 ## Stream Limit Increment {#stream-limit-increment}
 
-An endpoint limits the number of concurrently active incoming streams by
-limiting the maximum stream ID.  An initial value is set in the transport
-parameters (see {{transport-parameter-definitions}}) and is subsequently
-increased by MAX_STREAM_ID frames (see {{frame-max-stream-id}}).
+An endpoint limits the number of concurrently active incoming streams.  An
+initial value is set in the transport parameters (see
+{{transport-parameter-definitions}}) and is subsequently increased by
+MAX_STREAMS frames (see {{frame-max-streams}}).  Separate limits apply to
+bidirectional and unidirectional streams.
 
 As with stream and connection flow control, this document leaves when and how
-many streams to make available to a peer via MAX_STREAM_ID to implementations,
-but offers a few considerations.  MAX_STREAM_ID frames constitute minimal
-overhead, while withholding MAX_STREAM_ID frames can prevent the peer from using
-the available parallelism.
+many streams to make available to a peer via MAX_STREAMS to implementations.
+Implementations might choose to increase limits as streams close to keep the
+number of streams available to peers roughly consistent.
 
-The STREAM_ID_BLOCKED frame ({{frame-stream-id-blocked}}) can be
-used to signal a shortage of available streams. Implementations will likely
-want to increase the maximum stream ID as peer-initiated streams close.
+The STREAMS_BLOCKED frame ({{frame-streams-blocked}}) signals that a new stream
```

Comment:
Ugh.  So we now have STREAM_BLOCKED and STREAMS_BLOCKED?  No one is *ever* going to get those backward, no....

Reply:
I had exactly the same thought.  It's just that I can't think of a better name.

Reply:
Perhaps, in the interest of consistency....
- MAX_DATA => DATA_BLOCKED
- MAX_STREAMS => STREAMS_BLOCKED
- MAX_STREAM_DATA => STREAM_DATA_BLOCKED

Reply:
\o/ thanks.